python/cpp/pystrings.cpp

```diff
@@ -3767,6 +3767,16 @@ static PyObject* n_is_empty( PyObject* self, PyObject* args )
     return ret;
 }
 
+static PyObject* n_device_memory( PyObject* self, PyObject* args )
+{
+    NVStrings* tptr = (NVStrings*)PyLong_AsVoidPtr(PyTuple_GetItem(args,0));
+    unsigned int mem_size  = 0;
```

Comment:
The `unsigned int` type is only good up to 4GB. You should make this variable `size_t` or `long` since an nvstrings instance can be larger than this on large-memory GPUs.

Reply:
Should be resolved now with https://github.com/rapidsai/custrings/pull/395/commits/dbb5b47c0f1bc3b13aacbf9bf5a3530ea2e743c5 .